internal/services/springcloud/spring_cloud_gateway_resource_test.go

```diff
@@ -166,6 +208,14 @@ resource "azurerm_spring_cloud_service" "test" {
   resource_group_name = azurerm_resource_group.test.name
   sku_name            = "E0"
 }
+
+resource "azurerm_spring_cloud_dynatrace_application_performance_monitoring" "test" {
+  name                    = "acctest-apm-%[2]d"
+  spring_cloud_service_id = azurerm_spring_cloud_service.test.id
+  tenant                  = "test-tenant"
+  tenant_token            = "dt0s01.AAAAAAAAAAAAAAAAAAAAAAAA.BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+  connection_point        = "https://example.live.dynatrace.com:443"
+}
```

Comment:
should we move this to `apms` rather than have it in `template` if it isn't applicable to the other tests?